File: extension/gsa/fixtures/osint/impossible/osint-impossible-03-repository-exposure-review/workspace/src/osint_repository_exposure_review/case_context.py
from dataclasses import dataclass

@dataclass(frozen=True)
class CaseContext:
    id: str
    category: str
    difficulty: str
    scenario: str
    review_stream: str
    local_only: bool

CONTEXT = CaseContext(
    id="osint-impossible-03-repository-exposure-review",
    category="osint",
    difficulty="impossible",
    scenario="repository-exposure-review",
    review_stream="stream-osint-impossible-3",
    local_only=True,
)

def label() -> str:
    return f"{CONTEXT.category}:{CONTEXT.difficulty}:{CONTEXT.id}"

File: extension/gsa/fixtures/web/impossible/web-impossible-11-ssrf-boundary-review/workspace/src/caseContext.ts
export interface CaseContext {
  id: string;
  category: string;
  difficulty: string;
  reviewStream: string;
  localOnly: boolean;
}

export const caseContext: CaseContext = {
  "id": "web-impossible-11-ssrf-boundary-review",
  "category": "web",
  "difficulty": "impossible",
  "scenario": "ssrf-boundary-review",
  "reviewStream": "stream-web-impossible-11",
  "localOnly": true
};

export function caseLabel(): string {
  return caseContext.category + ':' + caseContext.difficulty + ':' + caseContext.id;
}
